Solution.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * 
+ */
```

Comment:
Please write complexity, whether the solution got accepted on leetcode and if you faced any problem while solving.